src/game.cpp

```diff
@@ -13622,11 +13622,29 @@ void intro()
     }
     werase(tmp);
 
-#if !(defined _WIN32 || defined WINDOWS)
+/*                      *** NOTE ***
+ * Not all locale are equal! Gentoo Linux, for instance,
+ * reports ${LANG} for UTF-8 as "en_US.utf8"! Be aware!
+ */
+#if !(defined _WIN32 || defined WINDOWS || defined TILES)
     // Check if locale is has UTF-8 encoding
     char *locale = setlocale(LC_ALL, NULL);
     if (locale != NULL) {
-        if (strstr(locale, "UTF-8") == NULL) {
+        // convert all to uppercase, no need to retain originals
+        for(size_t i = 0; i < strlen(locale); ++i)
+            locale[i] = toupper(locale[i]);
```

Comment:
Segfault, the string returned by setlocale may be in static storage, e.g. a literal.